cibuildwheel/macos.py

```diff
@@ -180,6 +238,41 @@ def setup_python(python_configuration: PythonConfiguration,
         # https://github.com/python/cpython/blob/a5ed2fe0eedefa1649aa93ee74a0bafc8e628a10/Lib/_osx_support.py#L260
         env.setdefault('ARCHFLAGS', '-arch x86_64')
 
+    if python_configuration.version == '3.9':
```

Comment:
Once we have access to packaging from #536 , then we should do a `>=` 3.9 here so we don't have to change it when we add 3.10. But this is fine until then.